server/http.js

```diff
@@ -35,12 +37,19 @@ app.use(log());
 
 // init placeholder
 console.error( 'loading data' );
-var ph = new Placeholder();
+var ph = new Placeholder({ readonly: true });
```

Comment:
this setting currently does nothing but I left it in because, although it doesn't actually do anything it's valid to open ph in read-only mode in server mode.

in the subsequent PRs this setting will case the underlying databases to be opened in read-only mode.

Reply:
good catch, this was introduced in https://github.com/pelias/placeholder/commit/8d83744e69dda00673412d2890f32cd5446899fb, I will add another commit to fix it in this PR